src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunGenericHelperNode.cs

```diff
@@ -49,6 +49,8 @@ private static GenericLookupResult GetLookupSignature(NodeFactory factory, Ready
                     return factory.GenericLookup.MethodDictionary((MethodDesc)target);
                 case ReadyToRunHelperId.VirtualCall:
                     return factory.GenericLookup.VirtualCall((MethodDesc)target);
+                case ReadyToRunHelperId.VirtualDispatchCell:
+                    return factory.GenericLookup.VirtualMethodAddress((MethodDesc)target);
                 case ReadyToRunHelperId.ResolveVirtualFunction:
```

Comment:
These need cleaning up. After this change, `VirtualCall` and `ResolveVirtualFunction` should never be used with the generic helper, but the `GenericLookupResults` are still named after them and we still have two dictionary cells. Some of this is reused in Project X because I missed when they reused the hacky cells to mean the thing we want them to be. Cleaning that up is a bit more involved and I'll do that in a separate change. I'm at least as disturbed by their existence as you are :)